BrowserKit/Sources/ComponentLibrary/Buttons/SecondaryRoundedButton.swift

```diff
@@ -5,16 +5,24 @@
 import Common
 import UIKit
 
-public class SecondaryRoundedButton: LegacyResizableButton, ThemeApplicable {
+public class SecondaryRoundedButton: ResizableButton, ThemeApplicable {
     private struct UX {
         static let buttonCornerRadius: CGFloat = 12
         static let buttonVerticalInset: CGFloat = 12
         static let buttonHorizontalInset: CGFloat = 16
         static let buttonFontSize: CGFloat = 16
+
+        static let contentInsets = NSDirectionalEdgeInsets(
+            top: buttonVerticalInset,
+            leading: buttonHorizontalInset,
+            bottom: buttonVerticalInset,
+            trailing: buttonHorizontalInset
+        )
     }
 
     private var highlightedTintColor: UIColor!
     private var normalTintColor: UIColor!
+    private var foregroundColorForState: ((UIControl.State) -> UIColor)?
```

Comment:
similar to the comment in this [PR](https://github.com/mozilla-mobile/firefox-ios/pull/17725#discussion_r1424414774), what do you think about refactoring to keep consistency between the two buttons? 

cc: @thatswinnie @lmarceau

Reply:
I've replied directly in the linked PR to keep the discussion in a single place